src/components/common/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./DropDownStyles.css";
+
+class DropDown extends React.Component {
+    render() {
+        let dropDownElems = [];
+
+        for (let i = 0; i < this.props.contents.length; i++) {
```

Comment:
as thinkster.io this the current best practice however I've changed this for loop to JavaScripts Map function.